ztp/source-crs/ClusterVersion.yaml

```diff
@@ -5,9 +5,12 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "100"
 spec:
-  # The default channel is stable-<version>
-  channel: $channel
-  # The default upstream path is https://api.openshift.com/api/upgrades_info/v1/graph
-  upstream: $upstream
-  # desiredUpdate:
-  #   version: $version
+  channel: stable-4.10
+  upstream: https://api.openshift.com/api/upgrades_info/v1/graph # default upstream.
+  desiredUpdate:
+    force: false
+    version: $version
+status:
```

Comment:
What will happen if we use this clusterverion.yaml to generate policy in enforce mode ? I think  this will cause issues as override the status is not allowed. 
May be better to add the status in the PGT so when we want to generate enforce policy we can remove it

Reply:
All requests for status changes are ignored so no issue with enforce policy has status field.